aim/web/ui/src/modules/BaseExplorer/components/BoxFullViewPopover/BoxFullViewPopover.tsx

```diff
@@ -181,34 +181,36 @@ function BoxFullViewPopover({
                           {groupConfigKey}
                         </Text>
                         <Text tint={70}>
-                          ({data.groups[groupConfigKey].items_count_in_group}
+                          ({data.groups[groupConfigKey].items_count_in_group}{' '}
```

Comment:
let's replace it to `({data.groups[groupConfigKey].items_count_in_group} items in this group)`

Reply:
It is written in that way @VkoHov, just formatted by `prettier`